test/Release/MysqlReleaseToggleCollectionGatewayTest.php

```diff
@@ -43,6 +43,7 @@ public function setUp()
 
     public function tearDown()
     {
+        $this->deleteAddedTogglesMarketingInformation();
```

Comment:
There's a LabsTest.php abstract class in labs-php-client-mysql. Probably you should use that in the feature, it uses a transaction and rolls it back at tearDown, so you wouldn't need to write code for undoing DB modifications.